db/schema.rb

```diff
@@ -638,6 +638,7 @@
     t.boolean  "exporting_photos",                                 default: false
     t.string   "color_theme",                        limit: 255
     t.boolean  "post_default_public",                              default: false
+    t.boolean  "chat_auto_login",                                  default: false
```

Comment:
Well, it depends how the "automatically log-in" is implemented here. IMO, what diaspora* should do is simply to remember your status between each page and each visit. It means, if last time I used diaspora*, I was online when I logged out, I should be online when I log back. Same for offline, or busy, or... Which means the default will only be used for the very first time the user will access the chat.